test/test_json.rb

```diff
@@ -0,0 +1,92 @@
+require_relative "helper"
+require "puma/json"
+
+class TestJSON < Minitest::Test
```

Comment:
For all of these, does it make sense to assert `JSON.parse(Puma::JSON.generate(value)) == value`?

Reply:
Ensuring that `Puma::JSON.generate` returns strings that can by parsed by `::JSON.parse` seems useful, for sure.

Updated in b905c34c264e847abed3b36d0711c95265deea2e